database/migration/migrateData.ts

```diff
@@ -1434,6 +1434,83 @@ const migrateData = async () => {
+    ALTER TABLE public.user ALTER COLUMN username TYPE citext;
+    `)
+
+    await DB.changeSchema(`
+    CREATE OR REPLACE VIEW user_org_join AS
+      SELECT
+        "user".id AS user_id,
+        username,
+        first_name,
+        last_name,
+        email,
+        date_of_birth,
+        password_hash,
+        organisation_id AS org_id,
+        name AS org_name,
+        user_role,
+        registration,
+        address,
+        logo_url,
+        is_system_org
+      FROM
+        "user"
+        LEFT JOIN user_organisation ON "user".id = user_id
+        LEFT JOIN organisation ON organisation.id = organisation_id;`)
+
+    await DB.changeSchema(`
+    CREATE OR REPLACE VIEW permissions_all AS (
+      SELECT
+          "user".username AS "username",
+          organisation.name AS "orgName",
+          "template".code AS "templateCode",
+          permission_name.name AS "permissionName",
+          permission_name.description AS "description",
+          template_permission.stage_number AS "stageNumber",
+          template_permission.level_number AS "reviewLevel",
+          template_permission.allowed_sections AS "allowedSections",
+          template_permission.can_self_assign AS "canSelfAssign",
+          template_permission.can_make_final_decision AS "canMakeFinalDecision",
+          template_permission.restrictions AS "restrictions",
+          permission_policy.name AS "policyName",
+          permission_policy.type AS "permissionType",
+          permission_policy.is_admin AS "isAdmin",
+          permission_policy.id AS "permissionPolicyId",
+          permission_policy.rules AS "permissionPolicyRules",
+          permission_name.id AS "permissionNameId",
+          template_permission.id AS "templatePermissionId",
+          "template".id AS "templateId",
+          "user".id AS "userId",
+          permission_join.id AS "permissionJoinId",
+          permission_join.organisation_id AS "orgId",
+          CASE WHEN template_category.ui_location @> (ARRAY['USER']::public.ui_location[]) THEN
+              TRUE
+          ELSE
+              FALSE
+          END AS "isUserCategory",
+          permission_name.is_system_org_permission AS "isSystemOrgPermission",
+          permission_join.is_active AS "isActive"
+      FROM
+          permission_name
+          LEFT JOIN permission_join ON permission_join.permission_name_id = permission_name.id
+          JOIN permission_policy ON permission_policy.id = permission_name.permission_policy_id
+          LEFT JOIN "user" ON permission_join.user_id = "user".id
+          LEFT JOIN organisation ON permission_join.organisation_id = organisation.id
+          LEFT JOIN template_permission ON template_permission.permission_name_id = permission_name.id
+          LEFT JOIN "template" ON "template".id = template_permission.template_id
+          LEFT JOIN template_category ON "template".template_category_id = template_category.id);
+    `)
```

Comment:
Now that all views are recreated in `43_views_functions_triggers.sql`, you don't have to recreate them here.

(Update from develop)